tests/controllers/test_collection_instrument_controller.py

```diff
@@ -242,3 +246,39 @@ def test_get_linked_cis_and_cir_version(self, get_response_json_from_service):
         ]
 
         self.assertEqual(result, expected)
+
+    def test_successful_delete_of_ci_and_cir_version_from_registry_table(self):
```

Comment:
Not sure about the naming for these tests. We're not deleting the collection instrument, we're deleting the registry instrument or the cir version.

Reply:
That's a good point. I became blindsided by all the similar terminology. I've updated.